### ISSUE TITLE
improvements in cisco jabber

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8248]
### Steps to reproduce:
I know there is a plugin developed for the jabber cisco, and I'm using this addom.
However even with the use of this add-on I have some suggestions that I would like to make to nvda to make your browsing easier.
In my opinion these changes could be made in the nvda, to eliminate the obligation of the complement, however if they think that these improvements should be carried out in the addom there are no problems.
Not listed in addoms add-ons because I do not feel very comfortable taking part in it, I do not speak English and I communicate with you using google translator.
Plus if anyone is willing to take this case the list would also be very good.
the courts:
Open the jabber.
Open the menu.
enter options.
Now nvda announces the object in focus as a panel.
Making use of jaws with the jabber compatibility scripts jaws informs list.
Based on the behavior of jaws I believe that nvda can do the same.
This is actually a list where you can change the categories of settings in jabber.

Another improvement:
Enter a chat with a person in the jabber.
Type a phrase.
Now walk with tabe to the conversation history.
you can now navigate through this history with the arrows up and down.
Again when doing this same test jaws recognizes this history field as a table and allows you to navigate

### Expected behavior:
When in the list within options in the jabber the nvda should inform list.
When in the conversation history, nvda should treat the object as a table.

### Actual behavior:
While in the list inside jabber options the nvda informs just panel, and not le when walking with arrows aboveor below.
When in the conversation history window nvda does not use the navigation keys in a table.

### System configuration:
NVDA version:
next 15103

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 7

Name and version of other software in use when reproducing the issue:
cisco jabber 11.9.3
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
no
I will put two logs with the information when in the two situations reported.
[jabber_history.txt](https://github.com/nvaccess/nvda/files/1981604/jabber_history.txt)
[jabber_options.txt](https://github.com/nvaccess/nvda/files/1981605/jabber_options.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva how is the behavior with last version of Jabber, last version of NVDA and last version of the addon developped by @dglee42?

**Comment by fernando-jose-silva** [comment]
Unfortunately I can no longer help with this issue, I no longer have access to jabber.
